sys/include/universal_address.h

```diff
@@ -28,7 +28,18 @@ extern "C" {
 #include <stdint.h>
 #include <stdlib.h>
 
-#define UNIVERSAL_ADDRESS_SIZE (16)         /**< size of the used addresses in bytes */
+/** @brief  size of the used addresses in bytes */
+/* determine the widest possible address type */
+#ifndef UNIVERSAL_ADDRESS_SIZE
+#define UNIVERSAL_ADDRESS_SIZE (0)       /* rather senseless default, should
```

Comment:
Well, the `#warning` is gcc specific and would trigger an error. Actually, I'm not sure, but I think I thought about this problem while creating the PR and the conclusion might have been that actually rising an error if this module is included without setting the value for its size **is** an error.